src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlClientEventSource.cs

```diff
@@ -739,12 +901,18 @@ internal void AdvancedScopeLeave(long scopeId)
             WriteEvent(AdvancedScopeExitId, scopeId);
         }
 
-        [Event(AdvancedTraceBinId, Level = EventLevel.Informational, Keywords = Keywords.AdvancedTraceBin)]
+        [Event(AdvancedTraceBinId, Level = EventLevel.Verbose, Keywords = Keywords.AdvancedTraceBin)]
```

Comment:
Same as above, if you intend this it's fine just want to check.

Reply:
Yes, intentional, should be Verbose.

Reply:
Ok, thanks.